bashflux/version.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func NewVersionCmd() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "version",
+		Short: "Get manager version",
+		Long:  `Mainflux server health checkt.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			Version()
+		},
+	}
+
+	return &cmd
+}
+
+// Version - server health check
+func Version() {
+	url := fmt.Sprintf("%s/http/version", serverAddr)
```

Comment:
Seems like you are using HTTP adapter version endpoint here? If that's the case, maybe you should use `version` endpoint of `things` or `users` service, not a protocol adapter.